tox.ini

```diff
@@ -10,10 +10,13 @@ deps =
 	# Ideally all the dependencies should be set as "extras"
 commands =
 	pytest {posargs}
-usedevelop = True
+usedevelop =
+    native: False
+    !native: True
 extras = testing
 passenv =
 	SETUPTOOLS_USE_DISTUTILS
+	PY_IGNORE_IMPORTMISMATCH
```

Comment:
Wow, I haven't heard of this thing. Is it documented anywhere?

Reply:
😅 digging issues: https://github.com/pytest-dev/pytest/issues/2042

Reply:
Oh, looks like my google-fu let me down, thanks for this link!